python/cuml/cuml/accel/tests/scikit-learn/xfail-list.yaml

```diff
@@ -1,4 +1,94 @@
-- reason: Test is failing with cuml.accel
+- reason: Test should fail with cuml.accel (scikit-learn 1.5)
+  marker: cuml_accel_bugs
+  condition: "scikit-learn<1.6"
```

Comment:
Question around this condition - do these tests pass in scikit-learn 1.6 (and if so, why?)?

Or do the tests not exist in scikit-learn 1.6? If it's the latter, would removing the condition cause the test run to fail in some way (so typos/stale test markers are detectable)? Or is the condition technically not necessary in that case?

Same with the `>= 1.6` case below.